pkg-config.log

```diff
@@ -0,0 +1,121 @@
+{"level":"info","ts":1591560832.1450043,"msg":"Started pkg-config","arg0":"/go/bin/pkg-config","args":["--libs","flux"]}
```

Comment:
should this file be committed? If not should we ignore it?

Reply:
Oops. That'll teach me to use a fancy GUI git client. No need to ignore it; it's only there because I explicitly created it while debugging the `pkg-config` issue.